app/Http/Controllers/Store/VoucherController.php

```diff
@@ -3,12 +3,133 @@
 namespace App\Http\Controllers\Store;
 
 use App\Http\Controllers\Controller;
+use Carbon\Carbon;
+use Illuminate\Support\Facades\Storage;
+use Log;
+use URL;
+use ZipArchive;
+use ZipStream\ZipStream;
 
 class VoucherController extends Controller
 {
-    // This belongs here becuase it's largely about arranging vouchers
+    // This belongs here because it's largely about arranging vouchers
     public function exportMasterVoucherLog()
     {
-        return response('', 200);
+        $dashboard_route = URL::route('store.dashboard');
+
+        // Setup the Storage dir
+        $disk = Storage::disk(config('arc.mvl_disk'));
+        $archiveName = config('arc.mvl_filename');
```

Comment:
Only nit picking... mixing camel case with underscore for var names in the same file - hinders reading a tiny bit.